core/src/main/java/bisq/core/support/dispute/arbitration/ArbitrationManager.java

```diff
@@ -618,12 +618,18 @@ public static MoneroTxWallet arbitratorCreatesDisputedPayoutTx(Contract contract
 
       // TODO (woodser): include arbitration fee
       //System.out.println("Creating feeEstimateTx!");
-      MoneroTxWallet feeEstimateTx = multisigWallet.createTx(new MoneroTxConfig()
-              .setAccountIndex(0)
-              .addDestination(buyerPayoutAddress, buyerPayoutAmount.multiply(BigInteger.valueOf(4)).divide(BigInteger.valueOf(5))) // reduce payment amount to compute fee of similar tx
-              .addDestination(sellerPayoutAddress, sellerPayoutAmount.multiply(BigInteger.valueOf(4)).divide(BigInteger.valueOf(5)))
-              .setRelay(false)
-      );
+      MoneroTxConfig moneroTxConfig = new MoneroTxConfig().setAccountIndex(0).setRelay(false);
+
+      BigInteger buyerAmount = buyerPayoutAmount.multiply(BigInteger.valueOf(4)).divide(BigInteger.valueOf(5)); // reduce payment amount to compute fee of similar tx
+      if (buyerAmount.compareTo(BigInteger.valueOf(0)) == 1) {
```

Comment:
would prefer ` > 0` check since it's more about being negative, 0, or positive (even though most comparators are implemented with -1, 0, 1)